spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Customer show page', type: :feature do
+  before(:each) do
+    @supermarket = Supermarket.create!(name: 'Bizarre Bazaar', location: '999 Oddity Ln.')
+    @customer = Customer.create!(name: 'Quirky Quinn')
+    @item1 = @supermarket.items.create!(name: 'Dragon Fruit', price: 10)
+    @item2 = @supermarket.items.create!(name: 'Phoenix Feather', price: 100)
+    @item = @supermarket.items.create!(name: 'Unicorn Horn', price: 1000)
+    @customer.items << [@item1, @item2]
+    visit "/customers/#{@customer.id}"
+  end
+  # User Story 1, cutomer show page
+  it 'shows the customer details' do
+    expect(page).to have_content('Quirky Quinn')
+  end
+
+  it 'shows the names and prices of all items' do
+    within 'ul.items' do
+      expect(page).to have_content('Dragon Fruit')
+      expect(page).to have_content('Phoenix Feather')
+      expect(page).to have_content('$10')
+      expect(page).to have_content('$100')
+      expect(page).to have_content('Bizarre Bazaar')
+    end
```

Comment:
I do like that you used a within block to really target where these values should appear. I also think we can be even more explicit, because with this setup, we don't know which item is associated with which price and which store. One way to do this is use within blocks tied to specific css ids that you give to each item. So `within("#item-1")` and then have the name, price, and store of item 1, then you could repeat that for item-2 to ensure everything is being displayed correctly.